From 6e180987d1f0aee0528370cd06e1a996a84bd8f7 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Wed, 27 Jul 2022 12:00:43 +0300
Subject: [PATCH] [SW-98081] gaudi2: disable sram bist

Due to couple of instabilities reported it is
decided to disable sram bist feature in 1.6.0.
This patch removes the call of sram self repair function.

Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
Change-Id: I61034fcaf2711306ff3cb2255fce36ca495dcc79
---

diff --git a/app/ppboot/ppboot.c b/app/ppboot/ppboot.c
index e454585..2866438 100644
--- a/app/ppboot/ppboot.c
+++ b/app/ppboot/ppboot.c
@@ -309,8 +309,6 @@
 	preboot_delay();
 	sram_set_extension();
 
-	hl_sram_self_repair();
-
 	ret = ppboot_load_preboot_img(&loadaddr);
 	ppboot_check_stack_monitor();
 	if (ret)
